source/manual/architecture-shallow-dive.html.md

```diff
@@ -0,0 +1,136 @@
+
+The user's browser uses [DNS](#dns) to figure out where to send the request, and sends that request to the Fastly content delivery network (CDN).
+
+If Fastly has the search results page cached, Fastly returns that page to the user.
+
+If Fastly does not have the search results page cached, Fastly sends the request on to Origin. Origin is the infrastructure that GOV.UK maintains, hosted in Amazon Web Services (AWS).
+
+Within Origin, the request goes to an application called Router to determine which GOV.UK application will produce the page.
+
+The Finder Frontend application serves search results pages, so Router sends the request to Finder Frontend.
+
+[Finder Frontend](https://github.com/alphagov/finder-frontend) renders finder and search pages for GOV.UK.
+
+Finder Frontend [queries the content store](https://www.gov.uk/api/content/search/all) for parts of the page content, such as the page title. Finder Frontend queries the Search API for the search results themselves. Search API provides a [JSON response with the search results](https://www.gov.uk/api/search.json?q=tax&count=2).
+
+Finder Frontend then renders the response in a search results page to the user.
+
+## A publisher publishes content on GOV.UK
+
+GOV.UK has the following applications to publish content:
+
+- Mainstream Publisher
+- Whitehall
+- Content Publisher
+- Specialist Publisher
+- Travel Advice Publisher
+- Collections Publisher
+
+The following sections describe what happens when a user publishes content to live and to draft using Mainstream Publisher (referred to as "Publisher").
+
+### A user publishes content to live
+
+A user signs into their Signon account. The user creates or changes content in Publisher and [saves that content in draft][#a-user-publishes-content-to-draft].
+
+When the user is ready to publish the content, they press __Publish__ in Publisher, which sends a request to the Publishing API.
+
+The [Publishing API](https://github.com/alphagov/publishing-api) publishes content to the Content Store. This API stores all versions of content, and performs validation checks whenever the API receives a new version.
+
+Publishing API updates the Content Store directly, which in turn updates Router to say there is now content at a particular URL, and which frontend application will render that content.
+
+Publishing API also puts the publishing event on a 'message queue', which a number of other applications listen to and use. For example:
+
+- the Search API listens to this queue and updates its indexes when new content is published so that this content can be searched for by users
+- the Email Alert Service listens to this queue so that it can send emails to users who have subscribed to that topic
+
+### A user publishes content to draft
+
+So far, we've been talking about the user-facing live stack. Before publishing content to live, a user can preview their content in the draft stack.
```

Comment:
Might be worth clarifying here that this means we actually have two origins (I appreciate this probably wasn't in the docs you read beforehand!):

```suggestion
So far, we've been talking about the user-facing live stack ("Origin"). Before publishing content to live, a user can preview their content in the draft stack ("Draft origin").
```